### ISSUE TITLE
create full display for command input

### ISSUE THREAD

**Issue by arlopurcell** [arloherrine/samurai#10]
This should include highlighing which card is to be played and its destination(s). The primary card and target (card/house/player) highlight should be consistent across all actions, plus a third color for secondary targets (I think the only one is where a spy is putting something after they steal it)

It might even be nice to have arrows, but that'll be a separate issue and might not be necessary, depending on how clear this issue makes things

depends on #8
